src/components/content/index.tsx

```diff
@@ -28,24 +24,11 @@ const options = {
       return <li>{children}</li>
     },
     [INLINES.HYPERLINK]: (node: any, children: any) => {
-      if (node.data.uri.includes("youtube.com/embed")) {
-        return (
-          <iframe

```

Comment:
Do I understand correctly that it is no longer possible to iframe videos? I didn't see it move to somewhere else. Since this was still used in the content pages afaik

Reply:
I removed it for privacy reasons, there was one page using it (the video is now linked)